src/components/GoodList.tsx

```diff
@@ -0,0 +1,11 @@
+export const GoodList: React.FC<{ goods: string[] }> = ({ goods }) => (
+  <>
+    <ul>
+      {
+        goods.map((good) => (
+          <li data-cy="Good" key={good}>{good}</li>
+        ))
+      }
+    </ul>
+  </>
```

Comment:
```suggestion
    <ul>
      {goods.map((good) => (
          <li data-cy="Good" key={good}>{good}</li>
      ))}
    </ul>
```
You don't need react fragment here, as you have ul wrap